### ISSUE TITLE
Missing binaries for Node 16, module version 93

### ISSUE THREAD

**Issue by chrisl8** [royaltm/node-murmurhash-native#25]
Node 16 has gone into LTS, but there are no binaries for it in the release, so using this package with Node 16 requires building locally.

**Comment by royaltm** [comment]
Yep. The problem is that node-pre-gyp-github is no longer working and I need to move to github Actions from Travis CI. Any help would be useful e.g. with examples on publishing binaries on github via github actions.

**Comment by chrisl8** [comment]
I will add this to my To Do list to dig into it. I know absolutely nothing about Github Actions personally, but I need to learn about it and I have people I can ask for help.

**Comment by chrisl8** [comment]
This weekend I moved a project of mine from using Travis CI to Github Actions for build testing, so now I can say that I have:

1. Used Github Actions.
2. Translated a project from Travis CI to Github Actions.

I will try to take a look at the Travis CI setup of this project sometime and try my hand at building a working Github Actions replacement . . . sometime.

**Comment by royaltm** [comment]
The goal is to make github action script that will not only build native modules but also publish them (make them downloadable from) as github releases and allow users to get them when they `npm install` the module.

**Comment by stouch** [comment]
If you install the lib inside a dockerfile, check that python is installed before `npm ci`
```
RUN apk add --no-cache python3 make g++ && rm -rf /var/cache/apk/*
RUN npm ci
```

And if you meet this error while a gitlab-runner execution on a kubernetes cluster, I think you can refer to this : https://gitlab.com/gitlab-org/gitlab/-/issues/345402#note_1053968530

**Comment by pencilcheck** [comment]
Getting error due to using node v21 and heroku keep attempting to download from binary at: https://github.com/royaltm/node-murmurhash-native/releases/download/v3.5.0/murmurhash-v3.5.0-node-v120-linux-x64.tar.gz

The build also error out
```
command "/app/.heroku/node/bin/node" "/app/.heroku/node/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/app/apps/server/node_modules/murmurhash-native/lib/Release/murmurhash.node" "--module_name=murmurhash" "--module_path=/app/apps/server/node_modules/murmurhash-native/lib/Release" "--napi_version=9" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v120
```
